tests/Tests/Utils.hs

```diff
@@ -34,8 +34,8 @@ withTempFile :: (FilePath -> Handle -> IO a) -> IO a
 withTempFile = bracket (openTempFile "." "crashy.txt") cleanupTemp . uncurry
   where
     cleanupTemp (path,h) = do
-      open <- hIsOpen h
-      when open (hClose h)
+      closed <- hIsClosed h
+      unless closed $ hClose h
```

Comment:
Why do we need this change? Seems equivalent to what was before?..

Reply:
I was having problems with windows and the semi-closed handle state. I'll try again without.

Reply:
This function will fail if the file is in a semi-closed state when doing lazy IO with LazyText and the temporary file will not be removed.

Reply:
Fair enough, thanks for checking.